pkgs/stdenv/generic/setup.sh

```diff
@@ -1053,23 +1065,26 @@ checkPhase() {
     if [[ -z "${checkTarget:-}" ]]; then
         #TODO(@oxij): should flagsArray influence make -n?
         if make -n ${makefile:+-f $makefile} check >/dev/null 2>&1; then
-            checkTarget=check
+            checkTarget="check"
```

Comment:
the RHS of variables don't need to be quoted

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2209